spec/spec_helper.rb

```diff
@@ -3,7 +3,7 @@
 require 'openstates'
 require 'pry'
 require 'pry-remote'
-require 'pry-debugger'
+require 'pry-byebug'
```

Comment:
I'll have to change the houndci settings for this repo, my preference is the exact opposite of this comment :smile:.